### DISCUSSION TITLE
Tech Stack Outline

### DISCUSSION THREAD

**Discussion by Fir121** [Fir121/nlp-project#2] [General]
To finalize the following:

- Frontend
- Backend
- Database

**Comment by prathamesh8** [comment]
**Brief Tech Stack**

**Backend**
- Framework: Django (for built-in authentication and ORM capabilities)
- Database: MySQL (for storing user authentication data and essay grading information)

**Frontend:**
- JavaScript Framework: React (for building dynamic user interfaces)
- CSS Framework: Bootstrap (for styling and layout)
- State Management: Redux (for complex state management, if needed)

**Version Control:**
- GitHub (for collaborative development, version control, and code review)

**Infrastructure Provisioning and Configuration:**
- Ansible (for provisioning and configuring infrastructure)
